### ISSUE TITLE
Staging Review finding: Conditionally revealed field already has error state

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#103789]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Dependent Experience Team
**Product:** 21-686c and 21-674
**Feature:** Critical Form Updates

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/current-marriage-information/type-of-marriage](https://staging.va.gov/view-change-dependents/add-remove-form-21-686c-v2/current-marriage-information/type-of-marriage) for the question "How did you get married?" when I select "Some other way" the conditionally revealed field is already displaying an error message when it first appears.

This is bad form for all users (I haven't done anything yet and you're telling me I've done it wrong!") but it's especially likely to confuse screen reader users, who may hear the error message "You must provide a response" before they hear the label on the new field. Without the visual cue that the new field has appeared, it's easy to interpret that error message as being for the "How did you get married" question that you just answered, and screen reader users may guess that something went wrong on that question.

<details><summary>Link, screenshot or steps to recreate</summary>

!["How did you get married?" question with "Some other way" selected and a new field "Other type of marriage" beneath it. "Other type of marriage" is blank but has the error message "You must provide a response.")

</details>

## Recommended action

I'm guessing this is a bug! Not sure what happened here, but the error message should only appear if a user fails to provide a response, not if they just haven't entered anything yet.

Note that I'm testing with test user +24, which did have an in-progress form that I've been editing. I'm wondering if this is happening because the user had already progressed further through the form?

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
